XcodeServerSDK/Utils/JSON.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 public protocol JSONSerializable {
-    init?(json: NSDictionary)
+    init?(json: NSDictionary) throws
```

Comment:
Ok; first things first; I'll never issue a `PR` that doesn't compile; that wouldn't make much sense.

Now; `non-throw functions` are a subtype of `throw functions` which means that whenever you expect a `throw function` you can pass a `non-throw function` and the compiler won't complain and code would work as expected without throwing.

The change in the protocol only opens the possibility to make the `init?(json: NSDictionary)` a `throwable` one (also we should drop the `?` since now it will either succeed or throw`) but all code that's not providing a `throws` on that init will work as it current does.

Here's a [blog post](http://robnapier.net/throw-what-dont-throw) that talks about «Throwing what doesn't throw»